entries/b/betking.com.json

```
@@ -0,0 +1,15 @@
+{
+  "Betking": {
+    "domain": "betking.com",
+    "tfa": [
+      "totp"
```

```suggestion
      "sms"
```

The two-factor authentication method described in the article is SMS, not TOTP